Doc/tutorial/datastructures.rst

```diff
@@ -676,7 +676,7 @@ to a variable.  For example, ::
    'Trondheim'
 
 Note that in Python, unlike C, assignment inside expressions must be done
-explicitly with the walrus operator ``:=``. This avoids a common class of
+explicitly with the `walrus operator`_ ``:=``. This avoids a common class of
```

Comment:
Since this links to another place in the same documentation, you should be able to just use a `:ref:` instead of having to hard code the whole link.

```suggestion
explicitly with the :ref:`walrus operator <why-can-t-i-use-an-assignment-in-an-expression>` ``:=``. This avoids a common class of
```

Reply:
Agreed (with a couple of line breaks).  Using a proper `ref` also means the backport should be correct and painless.

Reply:
Thanks, @csabella. I did not know that.